backend/api_v2/serializers.py

```diff
@@ -44,6 +45,12 @@ class Meta:
                 "This API name is already in use. Please select a different name."
             ),
         },
+        "unique_workflow_per_organization": {
```

Comment:
```suggestion
        "unique_api_per_workflow": {
```
NIT: The name is misleading. Also the above 2 values are repeated - check if its an issue we made with error handling
```
This API name is already in use. Please select a different name
```

Reply:
Removed this.